mwctl/Makefile

```diff
@@ -0,0 +1,38 @@
+
+export GOPATH=$(CURDIR)/.build
+
+GO ?= /usr/bin/go
```

Comment:
Yeah, I thought about this, which is why it is `?=`, so that you can do `GO=/usr/local/bin/go make darwin`.

I'm of two minds here: 

The Neck Beard in me says that this is a compiler, that it's not uncommon for people to have multiple versions installed, and the one in `PATH` might not be the Right One.  Basically, that explicit >> implicit.

The pragmatist in me however realizes that I might have to defend this principal to everyone who (understandably) is irritated by having to type an extra `GO=...` on each invocation.

So perhaps you are right and we should use `which`. :)